src/ReferenceWidget.vue

```diff
@@ -0,0 +1,125 @@
+<template>
+	<div>
+		<div v-if="reference && hasCustomWidget" class="widget-custom">
+			<div ref="customWidget" />
+		</div>
+
+		<div v-else-if="noAccess" class="widget-access">
+			<p>You may not have access to this reference!</p>
+		</div>
+
+		<a v-else-if="reference && reference.openGraphObject && !hasCustomWidget" :href="reference.openGraphObject.link" class="widget-default">
+			<div v-if="reference.openGraphObject.thumb" class="widget-default--image" :style="{ 'backgroundImage': 'url(' + reference.openGraphObject.thumb + ')' }" />
```

Comment:
Can we have something like a compact mode where the thumbnail is on top of the details?
Would be helpful e.g. when talks chat is in the sidebar?

Reply:
@jancborchardt Any thoughts about the compact mode? Maybe we reduce the amount of lines for the description maybe to 1? Shall we drop the preview for that, as adding it to the top will probably make the whole container quite large again.

Reply:
From my POV a preview is nice, especially for file shares and image links. I was more concerned about the width because taking half of that for the preview and half of it for the title/description it doesnt work good anymore

Reply:
Switched to still use side by side but dynamically adjust the rendered description lines

Reply:
![Bildschirmfoto vom 2022-08-31 14-22-00](https://user-images.githubusercontent.com/213943/187677201-9149988a-0ace-49b9-a778-871be7f08a74.png)

Where do I need to fetch and rebuild? 😄

Reply:
## Talk in files sidebar

Similar issue when Talk is used in the files sidebar:
![Bildschirmfoto vom 2022-08-31 14-27-47](https://user-images.githubusercontent.com/213943/187678175-007530a2-cb07-4f2e-a214-c2e8269b16d1.png)